docs/datamodel/constraints.rst

```diff
@@ -4,26 +4,112 @@
 Constraints
```

Comment:
Add a note that this page assumes basic edgeql knowledge to stay consistent with computeds?